src-ts/lib/modals/confirm/ConfirmModal.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC } from 'react';
+import Modal, { ModalProps } from 'react-responsive-modal';
+import { Button } from '../../button';
```

Comment:
these imports aren't grouped correctly and will fail the lint check.

if you set up your per the instructions i am about to send you, this and all auto-fixable lint issues should be fixed when you save the file.